### ISSUE TITLE
Option to turn mp4s into gifs?

### ISSUE THREAD

**Issue by mpred04** [nerezo/giftubebot#4]
Would there be any way to get the videos actually converted into gifs and posted within the same bot? I think that could be an incredible addition

**Comment by nerezo** [comment]
Actually the results are not in gif format. They are mp4 files so the gif files are needs more space than mp4 files.

**Comment by mpred04** [comment]
Yes, I understand, all I'm saying this would be interesting to have, if possible
